.gitattibutes

```diff
@@ -0,0 +1,2 @@
+dist/** binary
```

Comment:
Initially I thought "I love this", but then doesn't this make it hard/impossible to commit WIP for the dist files? Sometimes it's useful to commit partial work to unblock other team members. Did you consider this use case. We could just stash everything else and recompile, but I wonder what we're actually gaining by making this a `binary`.

Reply:
I didn't think of this use-case and personally it kinda sounds like a recipe for disaster; I'm happy to drop this whilst we decide though.

Reply:
Spoke with @shakyShane over MM and he thought this is ok, going to mark this as resolved.